packages/server/src/sdk/app/rows/queryUtils.ts

```diff
@@ -110,6 +110,9 @@ export const getQueryableFields = async (
     "_id", // Querying by _id is always allowed, even if it's never part of the schema
   ]
 
+  if (fields === undefined) {
```

Comment:
since this is an exported function, should it check for both `null` and `undefined`? A `fields == null` might be safer incase a caller were to do something like: `const fields: string[] | null = null` and then pass this